.eslintrc

```diff
@@ -5,7 +5,7 @@
     "node": true
   },
   "extends": ["next", "next/core-web-vitals", "prettier"],
-  "plugins": ["@typescript-eslint"],
+  "plugins": ["@typescript-eslint", "react-hooks"],
   "parser": "@typescript-eslint/parser",
   "settings": {
     "react": {
```

Comment:
could you revert this?
I think this plugin is already included in the next plugin

Reply:
both please 🙇